pipelines/nextflow/workflows/nextflow.config

```diff
@@ -1,32 +1,56 @@
-/* 
+/*
 See the NOTICE file distributed with this work for additional information
 regarding copyright ownership.
 
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-http://www.apache.org/licenses/LICENSE-2.0
+    http://www.apache.org/licenses/LICENSE-2.0
 
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-*/
 
-/*
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Nextflow config file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Default config options for all compute environments
-----------------------------------------------------------------------------------------
-*/
+-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-    Nextflow config file
+-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-    Default config options for all compute environments
+-----------------------------------------------------------------------------------------
+-*/
+
+NXF_WORK = System.getenv("NXF_WORK") ?: "$PWD/work"
 
 // Set minimum required Nextflow version, stopping execution if current version does not match
-//nextflowVersion = '!>=23.01'
+nextflowVersion = '!>=23.10'
```

Comment:
if it works with the default version in the cluster I would relax this condition

Reply:
The default we have (as standard after logging into codon) 
`nextflow -version` -> `version 24.10.3`
Also, given we want to use minimum nf-schema version 2.2.0 which requires nextflow version 23.10+